style.css

```diff
@@ -0,0 +1,306 @@
+/* global style */
+
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-size: inherit;
+  font-family: 'Roboto', sans-serif;
+  color: #fff;
+}
+
+:root {
+  --primaryColor: #fff;
+  --secondaryColor: #1f1f1f;
+}
+
+html {
+  font-size: 10px;
+}
+
+strong {
+  font-weight: 500;
+}
+
+.btn {
+  width: 24px;
+  height: 24px;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+
+input {
+  border: none;
+  outline: none;
+}
+
+.container {
+  min-height: 100vh;
+  background-color: var(--secondaryColor);
+}
+
+/* header style */
+
+header {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  height: 5.6rem;
+  background-color: #282828fa;
+  z-index: 69;
+}
+
+.menu-btn {
+  margin-right: 2.4rem;
+}
+
+.menu-btn svg {
+  fill: var(--primaryColor);
+}
+
+.home-btn {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 90px;
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+.home-btn svg {
+  fill: var(--primaryColor);
+}
+
+.search-bar {
+  width: 700px;
+  max-width: 700px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.btn .microphone {
+  height: 2.5rem !important;
+  width: 2.6rem !important;
+  fill: var(--primaryColor);
```

Comment:
Try to avoid adding too much `!important` in your code

Reply:
- [ ] I would recommend always including an explanation, a "why", for each requested change. For example:

This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

This request lacks a "why": Why don't you use an article tag here instead of a div?

Similarly, you have asked to avoid using much `!important` in your code but you have not explained why to avoid this. Please include an explanation as well.

Reply:
@Dipeshtwis thanks for the feedback, I have updated my comment : https://github.com/benjp009/Microverse-Week2-Trials-CodeReview/pull/1#discussion_r740277583

Is that ok ?